tests/scicat_test.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+from scitacean.testing.docs import setup_fake_client
+
+import ess
+from ess.reduce.scicat import download_scicat_file
+
+
+class Client:
```

Comment:
Use https://scicatproject.github.io/scitacean/generated/modules/scitacean.testing.backend.fixtures.fake_client.html instead. (This is a pytest fixture)